src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -29,6 +30,31 @@ const containerStyles = emotion.css`
     ${getZIndex('banner')}
 `;
 
+const measureTiming = (name: string) => {
```

Comment:
What do you think about splitting this out into it's own file? I think that would make sense since this isn't core logic tied to the Braze banner, and would make it reusable. Even better perhaps we could extend [`initPerf`](https://github.com/guardian/dotcom-rendering/blob/main/src/web/browser/initPerf.ts) to fit our use case? What if we updated the `end` function returned by `initPerf` to return the performance data? Existing callsites wouldn't have to care about that (they just won't use the returned data). And then in this component we'd just need to extract the data we need. If it's a concern that the version of `end` from `initPerf` logs when we might now want it to, we could add an (optional) arg to `end` which prevents the logging. So the method signature would become:

```js
const end = (dontLog: boolean = false): PerformanceEntryList => { ... }
```

What do you reckon?

(I found the type definition for `PerformanceEntryList` [here](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.performance.html#getentriesbyname)).

Reply:
Great idea.

Would you have a preference for returning the PerformanceEntryList rather than returning the time taken?

Reply:
(I.e. processing the PerformanceEntryList to product the millisecond value integer)

Reply:
Hey sorry I somehow missed this comment before. I think the way you've got it right now where you do the processing in `measureTiming` and return the ms sounds good 👍  If we have a use case in future where we want something different that requires the whole `PerformanceEntryList` we can revisit then.